examples/vite/src/application.tsx

```diff
@@ -9,17 +13,49 @@ import {
   Icon,
   Footer,
   Container,
+  IconButton,
+  PhaseBanner,
 } from "@govie-react/ds";
 
 export function App() {
   return (
     <div style={{ display: "flex", flexDirection: "column", height: "100%" }}>
-      <Header />
+      <Header
+        serviceName="Example"
+        homeHref="www.gov.ie"
+        serviceHref="#"
+        homeAriaLabel="aria label"
+      />
       <Container>
         <Heading>Heading</Heading>
-        <Paragraph>Paragraph</Paragraph>
-        <Link href="#">Link</Link>
+
+        <PhaseBanner level="alpha">This is a pre-release version</PhaseBanner>
+
+        <Link
+          href="https://www.google.com"
+          external={true}
+          noUnderline={true}
+          noVisited={true}
+        >
+          Link
+        </Link>
+
```

Comment:
non-blocking: Can we remove these extra lines/spacing? If its intentional let me know.

Reply:
done